TLM/TLM/Lifecycle/Patcher.cs

```diff
@@ -14,8 +14,8 @@ namespace TrafficManager.Lifecycle {
     using TrafficManager.UI.Helpers;
 
     public static class Patcher {
-        internal const string HARMONY_ID = "de.viathinksoft.tmpe";
-        internal const string HARMONY_ID_PF = "de.viathinksoft.tmpe.pathfinding";
+        internal const string HARMONY_ID = "me.tmpe";
```

Comment:
I think this should be public.
Maybe we should expose this in TMPE.API as well?

Reply:
Hmm, good idea

Reply:
@kianzarrin any suggestions where? Which class (new or existing)?